packages/itwin/grouping-mapping-widget/src/widget/components/QueryBuilder.ts

```diff
@@ -166,110 +164,59 @@ export class QueryBuilder {
     propertyValue: Primitives.Value,
```

Comment:
`addRelatedProperty` should be `private`.

Reply:
Changed. Removed underscore prefixes from private functions as well